### ISSUE TITLE
DAC_Scrollable_Regions_Usability_01

### ISSUE THREAD

**Issue by Shiny-Penny** [ONSvisual/local-statistics#138]
"Embedded scrollable regions within the page can be difficult to navigate for voice activation users as they cannot be referenced by name to allow users to quickly navigate into the region to initialise the scroll functionality."

DAC solution: Avoiding the use of embedded scrollable regions where possible may benefit some users. Please also note that embedded scrollable regions are only navigable via the use of the keyboard if they are focusable, or if there is focusable content within the region. On this page, the regions are not focusable themselves, and they do not contain focusable content. 

However, all the information is able to be viewed without the necessity to scroll so this did not cause an issue. If more information were situated within the region, users would not be able to view it. 

It is therefore advisable to either make it focusable or remove the embedded scroll altogether.

Issues log, Priority - AAA&Use, row 17
Report pg 105
